### ISSUE TITLE
Multiple users to "own" the same dat.

### ISSUE THREAD

**Issue by martinheidegger** [beakerbrowser/hashbase#60]
Hypercloud has upload limits & a "remove button" _(which are awesome)_ but this means that someone could publish dat and then at a later point remove it. If another person wants to make sure that a certain dat stays alive even if another person decides to remove it then she might want to "upload" the same dat under her account in order to make sure that it stays available (and for the url to be correct). The current [`'This archive is already being hosted by someone else'`](https://github.com/beakerbrowser/hashbase/blob/e470ada4878028e2d404c5fb87ed20523d4d2d75/lib/apis/archives.js#L78) is counter-productive in that case.

Would it be a good idea _(worth a PR)_ to change this to "allowed"?

I saw that this is [`Temporary`](https://github.com/beakerbrowser/hashbase/blob/e470ada4878028e2d404c5fb87ed20523d4d2d75/lib/dbs/archives.js#L168) as of https://github.com/datprotocol/hypercloud/commit/aa350c4b7fe80870650f0a1a30262b3d90009dcd _(7 months ago)_. Why is that?

_Note: @pfrazee mentioned that this is to reduce complexity?
 https://github.com/datprotocol/hypercloud/issues/115#issuecomment-351434697_

**Comment by pfrazee** [comment]
Yeah, it's been a while since that decision was made but IIRC we did it to save time. There's a couple places where assuming each archive had one "owner" made the code easier to write.

We do want to change this, so we'd be happy to get a PR. It's just going to take a little bit of work to accomplish, so it's up to you. (Not terrible but a night or two, I'd figure.)
